src/termdict/merger.rs

```diff
@@ -39,15 +39,20 @@ impl<'a> TermMerger<'a> {
         }
     }
 
+    /// Iterator over (segment ordinal, [TermOrdinal]) sorted by segment ordinal
+    ///
+    /// This method may be called
+    /// if [Self::advance] has been called before
+    /// and [true] was returned.
```

Comment:
please check doc for Merger::matching_segments as I wrote the doc without writing the implem and I didn't want to rabbit hole in the code too much ;)